development/tools/jobs/kyma/tests_test.go

```diff
@@ -21,15 +21,50 @@ var tests = []struct {
 		},
 	},
 	{path: "application-connector-tests", image: tester.ImageGolangBuildpackLatest},
+	{path: "application-connector-tests", image: tester.ImageBootstrap20181204, suite: tester.NewGenericComponentSuite,
+		additionalOptions: []jobsuite.Option{
+			jobsuite.JobFileSuffix("generic"),
+			jobsuite.Since(releases.Release17),
+			jobsuite.Optional(),
+		},
+	},
 	{path: "application-gateway-tests", image: tester.ImageGolangBuildpackLatest},
+	{path: "application-gateway-tests", image: tester.ImageBootstrap20181204, suite: tester.NewGenericComponentSuite,
+		additionalOptions: []jobsuite.Option{
+			jobsuite.JobFileSuffix("generic"),
+			jobsuite.Since(releases.Release17),
+			jobsuite.Optional(),
+		},
+	},
 	{path: "application-operator-tests", image: tester.ImageGolangBuildpackLatest},
+	{path: "application-operator-tests", image: tester.ImageBootstrap20181204, suite: tester.NewGenericComponentSuite,
+		additionalOptions: []jobsuite.Option{
+			jobsuite.JobFileSuffix("generic"),
+			jobsuite.Since(releases.Release17),
+			jobsuite.Optional(),
+		},
+	},
 	{path: "application-registry-tests", image: tester.ImageGolangBuildpackLatest},
+	{path: "application-registry-tests", image: tester.ImageBootstrap20181204, suite: tester.NewGenericComponentSuite,
+		additionalOptions: []jobsuite.Option{
+			jobsuite.JobFileSuffix("generic"),
+			jobsuite.Since(releases.Release17),
+			jobsuite.Optional(),
+		},
+	},
 	{path: "asset-store", image: tester.ImageGolangBuildpack1_11},
 	{path: "compass-runtime-agent", image: tester.ImageGolangBuildpack1_11,
 		additionalOptions: []jobsuite.Option{
 			jobsuite.JobFileSuffix("tests"),
 		},
 	},
+	{path: "compass-runtime-agent", image: tester.ImageBootstrap20181204, suite: tester.NewGenericComponentSuite,
+		additionalOptions: []jobsuite.Option{
+			jobsuite.JobFileSuffix("tests-generic"),
+			jobsuite.Since(releases.Release17),
+			jobsuite.Optional(),
+		},
+	},
```

Comment:
Didn't you miss those two `connection-token-handler-tests` and `connector-service-tests`?

Reply:
True. I added those.